docs/source/03_tutorial/06_visualise_pipeline.md

```diff
@@ -96,3 +96,72 @@ kedro viz --load-file my_shareable_pipeline.json
 ```
 
 And this will visualise the pipeline visualisation saved as `my_shareable_pipeline.json`.
+
+
+## Visualise Plotly charts in Kedro-viz
+
+[Plotly](https://plotly.com/python/) is a free and open source python library that allows you to make interactive, publication quality graphs. With Plotly integration on Kedro-viz, you can output your interactive charts as part of your pipline visualisation. 
+
+Kedro-viz aims to help users communicate different aspects of thier workflow and with Plotly intergration, we take one step further in this direction to allow our users share thier data insights as well. 
+
+We have also used Plotly intergration to allow users to [visualise metrics from experiments](https://kedro.readthedocs.io/en/stable/08_logging/02_experiment_tracking.html?highlight=experiment%20tracking). 
+
+NOTE : Kedro's Plotly integration is currently only supported for [Plotly Express](https://plotly.com/python/plotly-express/) charts. 
```

Comment:
You can format this note nicely using:
the "```eval_rst```" formatting

.. note:: Kedro's Plotly integration is currently only supported for [Plotly Express](https://plotly.com/python/plotly-express/) charts. 

See e.g. https://raw.githubusercontent.com/quantumblacklabs/kedro/main/docs/source/14_contribution/02_developer_contributor_guidelines.md